Mathlib/Algebra/MonoidAlgebra/Basic.lean

```diff
@@ -960,6 +960,15 @@ def mapDomainAlgHom (k A : Type*) [CommSemiring k] [Semiring A] [Algebra k A] {H
 #align monoid_algebra.map_domain_alg_hom MonoidAlgebra.mapDomainAlgHom
 #align monoid_algebra.map_domain_alg_hom_apply MonoidAlgebra.mapDomainAlgHom_apply
 
+@[simps!]
+def mapDomainAlgEquiv (k A : Type*) [CommSemiring k] [Semiring A] [Algebra k A] {H : Type*}
```

Comment:
Can you add a docstring?

Reply:
Done!

I note that dropping `#lint` into the file flags this but our CI does not. Is this expected!?

Reply:
It was turned off for porting, we need to turn it back on...